prov/verbs/src/verbs_info.c

```diff
@@ -139,6 +140,12 @@ const struct verbs_ep_domain verbs_rdm_domain = {
 	.caps			= VERBS_RDM_CAPS,
 };
 
+const struct verbs_ep_domain verbs_dgram_domain = {
+	.suffix			= "-dgram",
```

Comment:
Conceptually, DGRAM EPs support should be part of the same domain as MSG EPs.  Why is a separate domain necessary?

Reply:
Unfortunately, the vebrs/MSG and verbs/DGRAM has different realizations of the CQ, CNTR. When verbs/DGRAM domain is opened, NS threads is started.
It is hard to re-implement all of this at the current step.

Btw, the `fi_info` entry passes to the `fi_domain` function. Should `fi_info::ep_attr::type` be ignored in this case?

Reply:
@shefty Is it possible to stay it as is at this step?
I'll re-implement after I end up my work on the two PR - #3268 and #3199